src/ecs/wizards/executeCommand.ts

```diff
@@ -0,0 +1,93 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ecsExecToolkitGuideUrl } from '../../shared/constants'
+import { createCommonButtons } from '../../shared/ui/buttons'
+import { createInputBox } from '../../shared/ui/inputPrompter'
+import { createQuickPick, DataQuickPickItem } from '../../shared/ui/pickerPrompter'
+import { localize } from '../../shared/utilities/vsCodeUtils'
+import { WIZARD_EXIT, Wizard, WIZARD_BACK } from '../../shared/wizards/wizard'
+import { EcsContainerNode } from '../explorer/ecsContainerNode'
+
+export interface CommandWizardState {
+    task: string
+    command: string
+    confirmation?: 'yes' | 'supress'
+}
+
+function createTaskPrompter(node: EcsContainerNode) {
+    const taskItems = (async function () {
+        const taskArns = await node.listTasks()
+        if (taskArns.length === 0) {
+            return []
+        }
+        // Filter for only 'Running' tasks
+        const runningTasks = (await node.describeTasks(taskArns)).filter(t => {
+            return t.lastStatus === 'RUNNING' && t.desiredStatus === 'RUNNING'
+        })
+        return runningTasks.map(task => {
```

Comment:
Nit: can be condensed to `return (await ....describeTasks.... ).filter(....filter block....).map(....map block....)`